pipelines/build/regeneration/jdk10_regeneration_pipeline.groovy

```diff
@@ -15,62 +17,97 @@ limitations under the License.
 */
 
 String javaVersion = "jdk10"
+//TODO: Change me
+String DEFAULTS_FILE_URL = (params.DEFAULTS_URL) ?: "https://raw.githubusercontent.com/M-Davies/openjdk-build/parameterised_everything/pipelines/defaults.json"
```

Comment:
Needs updating

Reply:
I'll be doing that just before the merge